content/applications/sales/sales/amazon_connector/setup.rst

```diff
@@ -1,104 +1,151 @@
-==================================
-Configure Amazon Connector in Odoo
-==================================
+==============================
+Amazon Connector configuration
+==============================
 
-Register your Amazon account in Odoo
-====================================
+Odoo allows users to register an Amazon seller account in the database, but the user **must** have
+a paid Amazon Seller account prior to completing the configuration.
+
+To learn how to set up a paid Seller account on Amazon, log in to Amazon, and select *Start a
+Selling Account* from the *Account & Lists* drop-down menu, located in the header. Once that process
+is complete, proceed to follow the instructions below to register and link that Amazon Seller
+account in Odoo.
+
+Register Amazon account in Odoo
+===============================
```

Comment:
It's not a registration like signing up for a free trial, an Amazon Seller account, etc.

```suggestion
Connect Amazon Seller account to Odoo
=====================================
```